Pilot/pilotCommands.py

```diff
@@ -256,14 +242,84 @@ def _installDIRAC(self):
       except (IndexError, ValueError):
         continue
     # At this point self.pp.installEnv should contain all content of bashrc, sourced "on top" of (maybe) os.environ
-    self.pp.diracInstalled = True
+
+  def _installDIRACpy3(self):
+    """ Install python3 version of DIRAC client
+    """
+
+    # 1. Download DIRACOS
+    # curl -O -L https://github.com/DIRACGrid/DIRACOS2/releases/latest/download/DIRACOS-Linux-$(uname -m).sh
+    try:
+      machine = os.uname().machine  # py3
+    except AttributeError:
+      machine = os.uname()[4]  # py2
+
+    # FIXME: we should have a (set of) different location(s), starting from CVMFS
+    if not retrieveUrlTimeout(
+        "https://github.com/DIRACGrid/DIRACOS2/releases/latest/download/DIRACOS-Linux-%s.sh" % machine,
```

Comment:
Do you want a way to specify the version of DIRACOS2? I assume we will at some point but maybe it can wait for now.

Reply:
maybe, but as you say, we can live with what's done here for the moment.